src/overview.md

```diff
@@ -187,9 +187,8 @@ Dart's compiler technology lets you run code in different ways:
   an ahead-of-time (AOT) compiler for producing machine code.
 
 * **Web platform**: For apps targeting the web,
-  Dart includes both a development time compiler (dartdevc) and
-  a production time compiler (dart2js).
-  Both compilers translate Dart into JavaScript.
+  Dart can compile for debugging or production.
+  Its web compiler translates Dart into JavaScript.
```

Comment:
Technically there are 2 ocmpilers though... another thought here is to keep the original wording, but remove the words in parenthesis from the original sentence?

Reply:
I made a different update. Let me know what you think.

Reply:
How about "Its web compilers translate Dart into JavaScript" (plural compilers vs singluar compiler?)

Alternatively, combining both sentences:
Web platform: For apps targeting the web, Dart can compile to JavaScript, both for development and production purposes.